aksel.nav.no/website/sanity/schema/documents/designsystem-forside.tsx

```diff
@@ -0,0 +1,341 @@
+          type: "object",
+          validation: (Rule) => Rule.required(),
+          fields: [
+            defineField({
+              title: "Tittel",
+              name: "title",
+              type: "string",
+              validation: (Rule) => Rule.required(),
+            }),
+            defineField({
+              title: "Beskrivelse",
+              name: "description",
+              type: "text",
+              validation: (Rule) => Rule.required(),
+            }),
+            defineField({
+              title: "Ikon",
+              name: "icon",
+              type: "string",
+              options: {
+                list: ["Palette", "Code"],
+              },
+              validation: (Rule) => Rule.required(),
+            }),
+            defineField({
+              title: "Lenke",
+              name: "link",
+              type: "url",
+              validation: (Rule) =>
+                Rule.required().uri({
+                  scheme: ["http", "https"],
+                  allowRelative: true,
+                }),
+            }),
+          ],
+        }),
+      ],
+      initialValue: [
+        {
+          title: "Start med design",
+          description: "Figma-filer og kreative arenaer",
+          icon: "Palette",
+          link: "/grunnleggende/introduksjon/kom-i-gang-med-figma",
+        },
+        {
+          title: "Start med utvikling",
+          description: "Installasjon og tips",
+          icon: "Code",
+          link: "/grunnleggende/introduksjon/kom-i-gang-med-kodepakkene",
+        },
+      ],
+    }),
+    defineField({
+      title: "Lagene i designsystemet",
+      name: "ds_layers_overview",
+      type: "object",
+      validation: (Rule) => Rule.required(),
+      fields: [
+        defineField({
+          title: "Tittel",
+          name: "title",
+          type: "string",
+          validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+          title: "Ingress",
+          name: "ingress",
+          type: "text",
+          validation: (Rule) => Rule.required(),
+        }),
+      ],
+      initialValue: {
+        title: "Et fleksibelt system",
+        ingress:
+          "Aksel designsystemet består av flere lag som kan brukes enkeltvis eller som en full pakke.",
+      },
+    }),
+    defineField({
+      title: "Endringslogg",
+      name: "ds_changelog",
+      type: "object",
+      validation: (Rule) => Rule.required(),
+      fields: [
+        defineField({
+          title: "Tittel",
+          name: "title",
+          type: "string",
+          validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+          title: "Ingress",
+          name: "ingress",
+          type: "text",
+          validation: (Rule) => Rule.required(),
+        }),
+      ],
+      initialValue: {
+        title: "Endringslogg",
+        ingress:
+          "Se siste endringer i kode. (Se alle endringer.)[/dev/designsystemet/endringslogg]",
```

Comment:
I think we decided to remove this link?

Reply:
Jo, det stemmer. Prøvde å endre det i Sanity, men fikk det ikke til å oppdateres. Mulig dette hjelper? Uansett skal det ikke ligge i initialValue.

```suggestion
          "Se siste endringer i kode.",
```